src/components/Payment/CreatePaymentForm.tsx

```diff
@@ -176,7 +176,7 @@ function PaymentCreationForm(props: Props) {
     const tx = await createTransaction([payment], {
       accountData: props.accountData,
       memo: federationMemo.type !== "none" ? federationMemo : userMemo,
-      minTransactionFee: isMultisigTx ? multisigMinimumFee : 0,
```

Comment:
Ohh, I didn't remember this. Looks like there is already something similar in place. The change of this PR would only be to select twice the smart fee if it's higher than the min. multi-sig fee, right?

Let's talk about it tomorrow.